https://github.com/Reference-LAPACK/lapack/pull/1154

From b73a7c9e4de884ea7550832cb28ad518bab0c178 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 14 Sep 2025 11:27:26 +0200
Subject: [PATCH] fix library names in `lapack64.pc`
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add substitutions to `lapack.pc.in`, in order to ensure that
the generated `lapack64.pc` file correctly references the index64
library and BLAS dependency.  It seems that other pkg-config templates
have been updated as part of #462 but this one was omitted.
As a result, finding a lapack64 via pkg-config gave non-index64
libraries.

Signed-off-by: Michał Górny <mgorny@gentoo.org>
---
 lapack.pc.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lapack.pc.in b/lapack.pc.in
index 316c871011..27cd99f7c8 100644
--- a/lapack.pc.in
+++ b/lapack.pc.in
@@ -5,5 +5,5 @@ Name: LAPACK
 Description: FORTRAN reference implementation of LAPACK Linear Algebra PACKage
 Version: @LAPACK_VERSION@
 URL: http://www.netlib.org/lapack/
-Libs: -L${libdir} -llapack
-Requires.private: blas
+Libs: -L${libdir} -l@LAPACKLIB@
+Requires.private: @BLASLIB@
